pennylane/pauli/pauli_arithmetic.py

```diff
@@ -217,18 +217,21 @@ def to_mat(self, wire_order, format="dense"):
 
         return reduce(kron, (matrix_map[self[w]] for w in wire_order))
 
-    def operation(self, wire_order=None):
+    def operation(self, wire_order=None, get_as_tensor=False):
         """Returns a native PennyLane :class:`~pennylane.operation.Operation` representing the PauliWord."""
         if len(self) == 0:
             if wire_order in (None, [], wires.Wires([])):
                 raise ValueError("Can't get the operation for an empty PauliWord.")
             return Identity(wires=wire_order)
 
         factors = [op_map[op](wire) for wire, op in self.items()]
+
+        if get_as_tensor:
+            return factors[0] if len(factors) == 1 else Tensor(*factors)
```

Comment:
Minor point: I see that `Tensor` is "[marked for deletion or overhaul](https://github.com/PennyLaneAI/pennylane/blob/e6f92efcbc2d8abcd9eec553b0c8dac3380b9a8b/pennylane/operation.py#LL1876C23-L1876C54)". Is there any alternative?

Reply:
Yes, but at that point Hamiltonians will be removed as well. Then we can just remove this line and in the tapering workflow, cast to operations instead 👍🏼